vagrant-pxe-harvester/README.md

```diff
@@ -17,22 +17,29 @@ Prerequisites
 -   vagrant-libvirt plugin \>= 0.0.43.
 -   KVM (i.e. qemu-kvm), preferably the latest and greatest. This
     environment was tested with qemu-kvm 2.11.
--   Host with at least 16 CPU, 64GB RAM, and 500GB free disk space.
+-   Host with at least 16 CPU, 64GB RAM, and 500GB free disk space. (note: running this is **very** resource intensive depending on the number of nodes)
 
 Quick Start
 -----------
 
 1.  Edit `settings.yml` to make sure the configuration satisfies your
     needs. The options are self-documented.
-2.  Run `setup_harvester.sh`. This may take awhile (i.e. 30 minutes
-    depending on configuration).
+    Set `harvester_cluster_nodes` to `1` if you just want to try out
+    Harvester quickly and don't need a full cluster.
+2.  Run `setup_harvester.sh` to deploy a Harvester cluster only.
+    This may take a while (i.e. 30 minutes  depending on configuration).
+    To deploy a Harvester cluster and a Rancher node, run `vagrant up`.
```

Comment:
```suggestion
    To deploy a Harvester cluster and a Rancher node, [enable Rancher](https://github.com/harvester/ipxe-examples/blob/08a7053e68e7983e3d1f6c776823f45c0b170261/vagrant-pxe-harvester/settings.yml#L141).
```

Reply:
I've opened https://github.com/harvester/ipxe-examples/pull/74 to fix that.